ssg/yaml.py

```diff
@@ -21,7 +21,17 @@
     from yaml import Loader as yaml_Loader
 
 try:
-    from yaml import CDumper as yaml_Dumper
+    # Regression with newer pyyaml where CDumper does not honor increase_indent
+    from yaml import CDumper as yaml_test_Dumper
+
+    class TestDumper(yaml_test_Dumper):
+        def increase_indent(self, flow=False, indentless=False):
+            return super(yaml_test_Dumper, self).increase_indent(flow, False)
+    unformatted_yaml = yaml.dump({"a": [1]}, None, TestDumper)
+    if unformatted_yaml == 'a:\n- 1\n':
+        from yaml import Dumper as yaml_Dumper
```

Comment:
This will have performance impact - we should prefer the CDumper.

Reply:
Ok, I'll drop that patch.